helm_deploy/dps-data-compliance/templates/_envs.tpl

```diff
@@ -0,0 +1,19 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Environment variables for web and worker containers
+*/}}
+{{- define "deployment.envs" }}
+env:
+  - name: SERVER_PORT
+    value: "{{ .Values.image.port }}"
+
+  - name: JAVA_OPTS
+    value: "{{ .Values.env.JAVA_OPTS }}"
+
+  - name: APPINSIGHTS_INSTRUMENTATIONKEY
```

Comment:
will this work?  Normally set to `APPLICATION_INSIGHTS_IKEY` I think

Reply:
It is correct as is, at least consistent with the name in AWS - this was following the offender-case-notes example